src/app/jobs/Job.js

```diff
@@ -8,50 +9,31 @@ export default class Job extends Logable {
     super();
 
     // you can customize below attributes of your jobs
-    this.QUEUE_NAME = 'default';
+    this.JOB_NAME = 'default';
```

Comment:
Can we use class name as job name by default ?
For example:
  MyJob class has job name 'MyJob'

It make developer more easily to debug their program.